about/release-notes/index.md

```diff
@@ -17,6 +17,78 @@ GitHub and be notified by email whenever a new release is available. On the
 click `Watch`, select `Custom` and then check `Releases`.
 </Highlight>
 
+
+## TimescaleDB&nbsp;2.11.0 on 2023-05-12
+
+This release contains new features and bug fixes since the 2.10.3 release.
+We deem it moderate priority for upgrading.
```

Comment:
There's a highlight block to use for this, in the template.

Reply:
added it; along with:
```
These release notes are for the release of TimescaleDB&nbsp;2.11.0 on
2023-05-22.
```

which seem redundant - but its there for earlier entries

Reply:
It's just a sentence to avoid having a title with no content under it. It helps to improve readability.